example/stream/main.cpp

```diff
@@ -18,7 +18,7 @@ static void MakeSpan(opentracing::Tracer& tracer, int span_index) {
 int main() {
   lightstep::LightStepTracerOptions options;
   options.collector_plaintext = true;
-  options.satellite_endpoints = {{"collector.lightstep.com", 80}};
+  options.satellite_endpoints = {{"localhost", 8360}};
```

Comment:
Should we include a link to instructions for setting up a local dev-mode satellite somewhere?

Reply:
Done